sirmordred/task_collection.py

```diff
@@ -147,6 +157,8 @@ def execute(self):
                          self.conf['es_collection']['url'],
                          self.conf[self.backend_section]['raw_index'])
 
+        return errors
```

Comment:
See line 89. I guess it should return errors, too. Please check any other place where the method returns.